app/src/main/java/com/checkin/app/checkin/menu/listeners/LocationSelectedListener.java

```diff
@@ -0,0 +1,5 @@
+package com.checkin.app.checkin.menu.listeners;
+
+public interface LocationSelectedListener {
+    void onSelectedLocation();
```

Comment:
And why the heck is this function without any parameters? I already told you that the location data will be sent as arguments to this function.